libs/isograph-react/src/tests/garbageCollection.test.ts

```diff
@@ -10,32 +11,36 @@ import {
 import { iso } from './__isograph/iso';
 import { nodeFieldRetainedQuery } from './nodeQuery';
 
-const getDefaultStore = () => ({
-  [ROOT_ID]: {
-    me: { __link: '0' },
-    you: { __link: '1' },
-    node____id___0: {
-      __link: '0',
+const getDefaultStore = (): IsographStore => ({
+  Query: {
```

Comment:
I want to generate `@isograph/react` :-P meaning we don't have to hard code `Query`, and can avoid emitting features that don't exist (e.g. if a given reader AST node is never generated, have a simpler `read.ts`). Eventually!